docs/specifications/tests/Delegation-TP/delegation03.md

```diff
@@ -39,21 +41,57 @@ response from the domain must fit into a 512 byte UDP packet.
       created above using the message compression schema defined
       in section 4.1.4 of [RFC 1035] where applicable.
 
-4. Obtain the name server IP addresses per name server names from 
-   the delegation from the parent using [Method4].
+4. The test case will test packet size twice in two different ways, 
+   and the packet created above (without the A and AAAA records) will 
+   be used a second time.
```

Comment:
This is not a step to be taken to execute the test case, right? It's just an explanation of what's going on. Wouldn't it be better to have a brief overview of the approach of the algorithm? E.g. in the objective section. That way the Steps could focus on getting things done.

Reply:
Indirectly it is a step to create a copy of the packet to be able to use the same packet twice.

Reply:
Could you make that instruction explicit? Or preferably add an explicit step where a second packet is created.

Reply:
Pushed a new solution.